website/content/en/docs/dev/internals.md

```diff
@@ -135,6 +135,10 @@ The directory contains the following files:
 - `$LIMA_SHELL`: `lima ...` is expanded to `limactl shell --shell ${LIMA_SHELL} ...`.
   - No default : will use the user's shell configured inside the instance
 
+- `$LIMA_TEMPLATES_PATH`: A list of directories to locate templates via
+  the `template://` schema.
+  - Default: the `/usr/local/share/lima/templates` directory.
+
```

Comment:
This should be migrated to https://lima-vm.io/docs/config/environment-variables/

Reply:
I'm slightly confused that most (but not all, e.g. not `LIMA_HOME`) are documented in both locations. What are the rules about what goes where?

Reply:
All the variables should be moved to https://lima-vm.io/docs/config/environment-variables/

Reply:
I guess I had just forgotten the existence of https://lima-vm.io/docs/dev/internals when I merged the PR for https://lima-vm.io/docs/config/environment-variables/

😞